### PR TITLE
add basic test for device request

### PR DESCRIPTION
**Pull request by Sasasu**
a part of #629

### CODE REVIEW

File: tests/regress/pl_schedule_py39 [hunk 1]
```diff
@@ -46,5 +46,9 @@ test: memory_parallel_python_1 memory_parallel_python_2 memory_parallel_python_3
 # PL/Container import pkg test
 #test: python3_import_module 
 
+# test case for PL/Container configuration
```
Comment (Sasasu) [inline tests/regress/pl_schedule_py39:49]: TODO: move to some `plcontainer server logic test` or somewhere...

File: tests/test_assign_gpu.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        raise RuntimeError("runtimeid '" + args.runtime + "' did not find in file '" + args.file + "'")
+
+    assign_gpu(runtime, args.action)
+
+    out_xml = xml_prettify(xmltree)
+
+    if args.i:
+        f = open(args.file, "w+")
+        f.write(out_xml)
+        f.close()
```
Comment (yihong0618) [inline tests/test_assign_gpu.py:53]: we can use `with` here?
```python
with open(args.file, "w+") as f:
    f.write(out_xml)
```

**Review by yihong0618 — APPROVED** [review]
LGTM

File: tests/regress/pl_schedule_py39 (hunk 1 above)
Comment (beeender) [reply tests/regress/pl_schedule_py39:49]: Add `TODO`/`FIXME` in the code, so we can grep it in the future. Put it in the PR comment is pretty useless ..

**Pull request merged**